server/ai_http.go

```diff
@@ -215,6 +216,34 @@ func (h *lphttp) LLM() http.Handler {
 	})
 }
 
+func (h *lphttp) StartVideoToVideo() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+		// skipping handleAIRequest for now until we have payments
+
+		// whats the point of openapi if we have to do this manually
```

Comment:
If you want to merge this PR, then please remove this comment.

Reply:
A bit of late-night frustration that crept through :) These handlers are largely boilerplate that should be done for us by codegen. Removed in https://github.com/livepeer/go-livepeer/commit/9e461a7d2ee59b1cd5603382d403c082b71469ad